File: jmoss6.java
//Julen Moss
//
//

public class jmoss6 {
	
	    public static void main(String[] args) {
	      
	        int x = 6, y = 12;
	        double a = 6.0, b = 12.0;

	        System.out.println("Integer evaluation:");

	        int result1 = x + y * x / y - x;
	        System.out.println("x + y * x / y - x = " + result1);

	        int result2 = -x - y / x * y + x;
	        System.out.println("-x - y / x * y + x = " + result2);

	        int result3 = x + y - x / y;
	        System.out.println("x + y - x / y = " + result3);

	        System.out.println("\nDouble evaluation:");

	        double result4 = a + b * a / b - a;
	        System.out.println("x + y * x / y - x = " + result4);

	        double result5 = -a - b / a * b + a;
	        System.out.println("-x - y / x * y + x = " + result5);

	        double result6 = a + b - a / b;
	        System.out.println("x + y - x / y = " + result6);
	    }
	}

File: jmoss6.c
//Julen Moss
//
//
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
int main(void) {
    int x = 6, y = 12;
    double a = 6.0, b = 12.0;

    printf("Integer evaluation:\n");

    int total1 = x + y * x / y - x;
    printf("x + y * x / y - x = %d\n", total1);
    
    int total2 = -x - y / x * y + x;
    printf("-x - y / x * y + x = %d\n", total2);
    
    int total3 = x + y - x / y;
    printf("x + y - x / y = %d\n", total3);

    printf("\nDouble evaluation:\n");

    double total4 = a + b * a / b - a;
    printf("x + y * x / y - x = %.2f\n", total4);
    
    double total5 = -a - b / a * b + a;
    printf("-x - y / x * y + x = %.2f\n", total5);
    
    double total6 = a + b - a / b;
    printf("x + y - x / y = %.2f\n", total6);
    return 0;
}
